Automattic/wp-calypso
# Gutenberg: Rename the selected editor actions names

#### Changes proposed in this Pull Request

In https://github.com/Automattic/wp-calypso/pull/33221 I renamed the actions for requesting / setting the selected editor from `EDITOR_TYPE_REQUEST`/`EDITOR_TYPE_SET` to `EDITOR_TYPE_FETCH`/`EDITOR_TYPE_RECEIVE`, because I thought it was more clear from a semantic point view, but according to @dmsnell apparently we're trying to use a `_REQUEST`/`_SET` terminology for this kind of actions, so we are renaming them on this PR.

#### Testing instructions

* With a site opted-in to Gutenberg, to go `/posts/:site` and start creating a new post or open an existing post.
* Make sure that you are redirected to the block editor.
* Now, switch to the classic editor from either the Inline Help or from the "More" menu item.
<img width="348" alt="Screen Shot 2019-06-11 at 10 02 53" src="https://user-images.githubusercontent.com/1233880/59236228-32328000-8c30-11e9-8a8c-a7ea369f29e3.png">
<img width="274" alt="Screen Shot 2019-06-11 at 10 03 15" src="https://user-images.githubusercontent.com/1233880/59236229-32328000-8c30-11e9-99a7-3a6042743726.png">

* Make sure you are redirected to the classic editor.
* Go back to the list of posts (`/posts/:site`).
* Create again a new post or open an existing post.
* Make sure you see the classic editor.
* Switch now again to the block editor from the sidebar.
<img width="266" alt="Screen Shot 2019-06-11 at 10 06 16" src="https://user-images.githubusercontent.com/1233880/59236319-92c1bd00-8c30-11e9-80bb-50f67ab901db.png">

* Make sure you're redirected to the block editor.

Fixes #33779
